### ISSUE TITLE
`lr_min`: the decay range is scaled from 1 to `lr_min` to ensure the learning rate does not drop below this minimum value.

### ISSUE THREAD

**Issue by unlimblue** [pytorch/torchtitan#1457]
https://github.com/pytorch/torchtitan/blob/f3e2a75e151872b3dc346d3bfa37d121e413242d/torchtitan/models/deepseek_v3/train_configs/deepseek_v3_671b.toml#L37

The term `lr_min` is defined as a scaling factor, not an absolute learning rate value. The current naming is highly misleading, and it's possible that even the official example `toml` files contain errors due to this confusion.

**​​Recommendation​​**: 

Rename it to a more explicit term, such as `min_lr_factor`.

**Comment by tianyu-l** [comment]
Sounds good to me. Could you submit a PR with references to the correct values?

**Comment by tianyu-l** [comment]
cc @wwwjn on deepseek training config
